layouts/shortcodes/blocks/partners-alternate.html

```diff
@@ -0,0 +1,27 @@
+{{ $_hugo_config := `{ "version": 1 }` }}
+{{ $col_id := .Get "color" | default .Ordinal }}
+{{ $height := .Get "height" | default "auto"  }}
+<section class="row td-box td-box--{{ $col_id }} td-box--gradient td-box--height-{{ $height }}">
+    <div class="container" style="max-width: 600px;">
+		<div class="row">
+			<div class="col mb-5 text-center">
+				<a href="https://github.com/tektoncd/friends"><img src="/partner-logos/tekton-friends.png" class="img-fluid"></a>
+			</div>
+		</div>
+    </div>
+    <div class="container">
+		<div class="row">
+			<div class="col mb-5 text-center">
+                <h3>Tekton is created by <a href="https://www.google.com/">Google</a>
+                    in partnership with <a href="https://www.alibabagroup.com/">Alibaba</a>, 
+                    <a href="https://www.antfin.com/">Ant Financial</a>,
+                    <a href="https://www.cloudbees.com/">CloudBees</a>,
+                    <a href="https://www.elastic.co/">Elastic</a>,
+                    <a href="https://www.ibm.com/">IBM</a>,
+                    <a href="https://puppet.com/">Puppet</a>,
+                    <a href="https://www.redhat.com/">Red Hat</a>,
+                    and <a href="https://triggermesh.com/">Trigger Mesh</a>.</h3>
```

Comment:
i like using the friends logo!! :D

some of these folks dont contribute to Tekton, but they are in tekton/friends as users - it would maybe be more safe to say something like "Tekton is created by the CDF" and leave it at that

Reply:
Done. I have rephrased it as "Tekton is a collaborative project where members of the ecosystem contribute together to make CI/CD easier for everyone. See the individuals and organizations that are involved in the Tekton project."

Reply:
Great!!! :D